src/components/Common/Avatar.tsx

```diff
@@ -22,21 +20,23 @@ type Props = {
   address?: string
   className?: string
   src?: string
+  avatar?: string
+  isLoadingDclProfile?: boolean
 }
 
-export default function Avatar({ address, size, src, className }: Props) {
-  const { profile, isLoadingProfile } = useProfile(address, !!src)
+// TODO: avatar should be mandatory when no src is provided
```

Comment:
`avatar` is the flat img of the user's in-world avatar (in case the user has one), or an image of a default avatar when user has no catalyst profile.

`src` is a link to an image (any image) which we are using for the forum avatars. 

So when the user has no custom avatar, we try to use the forum one.

```typescript
    {hasCustomAvatar ? (
            <Avatar address={address} avatar={avatar} size="medium" isLoadingDclProfile={isLoadingDclProfile} />
          ) : (
            <Avatar size="medium" src={avatarUrl} />
          )}
          ```
          
I think we could refactor this component in this PR :)

Reply:
AvatarFace is rendering an img too, even though it is adding some classes. Couldn't we just use AvatarFace in Avatar and remove the condition where an image is rendered?